README.md

```diff
@@ -99,3 +127,4 @@ console.log(i)
 
 -- YOUR ANSWER HERE --
 
+Mo  ?
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.